### ISSUE TITLE
2018.08 Report : Fee monitoring

### ISSUE THREAD

**Issue by ghost** [bisq-network/bisq-monitor-deprecated#11]
# 2018.08 Report : Fee monitoring

TL;DR; : Everything was OK with the Bisq dispatched fees this month. No issue.

I have (since july 24th) a python croned task monitoring the Bisq network onion nodes for the fees provided on the Bisq network. On a 5 minutes basis.
In august 2018, there were 48 alerts.
**All** alerts being false positives, ie simultaneously small gaps, and with no duration.

False positives are not too numerous, that's the reason (+ lack of time) why I didn't improve the code atm.
I prefer too numerous false positive than missing a real issue.

The monitoring is at the moment realized in two ways :
- writing alerts in a file
- sending an email (if a Bisq contributor is interested on also getting email alerts, please contact me.)

In case of, and since it doesn't cost much, the fees are also archived.

atm this monitoring runs on a computer protected by an UPS.
I may also install the monitoring on a raspberry and/or on a VPS. (But it must remain convenient.)

This monitoring is a subresult of the pricenode monitoring task (which is not finished ... but not far).
-> https://github.com/bisq-network/bisq-monitor/issues/7

If I'm on the wrong place for this report, apologies, and just tell me.

**Comment by cbeams** [comment]
> If I'm on the wrong place for this report, apologies, and just tell me.

This is the wrong place. There is a Monitoring Operator / Maintainer role at https://github.com/bisq-network/roles/issues/10.

I'm not sure what value this work provides if it's not providing alerts to @bisq-network/pricenode-operators. Ideally this would happen via messages coming into the #pricenode channel in Slack, which all operators should have notifications turned on for.

False positives have been a big problem in the past, and have been the reason previous monitoring solutions have been scrapped. Please make sure that what you're building is going to be of actual value before investing much more into it. I recommend engaging with the operators to see what they actually need and want. A good example would be: how could we have caught the BitcoinAverage outage and the stale prices that ensued earlier this week? I just happened to see it as it was occurring, but it would have been excellent to have an alert telling us that our pricenodes were off.

In any case, please close this issue. Thanks.

**Comment by ghost** [comment]
In fact, I did build the python code used for this fees monitoring as a first run before writing the code for the prices monitoring.
Since there are only 5 fees to monitor, the code is easier. That's the reason I begun with the fees.

I thought it may also be of use to have the fees archived.

Sending to slack is also on the todo list.

I close this issue here.
I'll reopen an issue at the appropriate place when my code for monitoring the pricenodes will be running.
